packages/calypso-e2e/src/hooks.ts

```diff
@@ -46,18 +46,29 @@ export const setupHooks = ( callback: ( { page }: { page: Page } ) => void ): vo
 		tempDir = await mkdtemp( path.join( resultsPath, sanitizedTestFilename + '-' ) );
 
 		// Start the browser
-		page = await start( {
-			logger: async ( name, severity, message ) => {
-				await appendFile(
-					path.join( tempDir, 'playwright.log' ),
-					`${ new Date().toISOString() } ${ process.pid } ${ name } ${ severity }: ${ message }\n`
-				);
+		await startBrowser( chromium );
+		// Launch context with logging.
+		const context = await newBrowserContext( {
+			logger: {
+				log: async ( name: string, severity: string, message: string ) => {
+					await appendFile(
+						path.join( tempDir, 'playwright.log' ),
+						`${ new Date().toISOString() } ${ process.pid } ${ name } ${ severity }: ${ message }\n`
+					);
+				},
+				// Default to verbose Playwright API logs.
+				isEnabled: ( name: string ) => name === 'api',
```

Comment:
I thought about it and tried implementing it, but gave up mostly because I couldn't get the helper method to work (for some reason). I will take another try at it though!

Reply:
Fixed in most recent revision.